src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod style;
 
 mod flexbox;
 mod forest;
+mod indexmap;
```

Comment:
This module should be feature-gated here.

Reply:
Done